lagotto/init.sls

```diff
@@ -1,123 +1,76 @@
-{% from 'lib/environment.sls' import environment %}
 {% from "lagotto/map.jinja" import props with context %}
+{% from "consul/lib.sls" import consul_service_domain, consul_service_definition %}
 
 {% set sidekiq_server = props.get('sidekiq_server', 'None') %}
 {% set oscodename = salt.grains.get('oscodename') %}
 
 include:
-  - nginx
   - lagotto.nginx
   - lagotto.common
-{% if grains['fqdn'] == sidekiq_server %}
-  - lagotto.sidekiq
-{% endif %}
+  - lagotto.lib.docker-common
 
-{% set app_root = props.get('app_root') %}
-{% set ruby_ver = props.get('version_ruby') %}
+{% set environment = salt.grains.get('environment') %}
+{% set app_name = 'lagotto' %}
```

Comment:
some of these assignments look like they're duplicated in `docker-common.sls`

Reply:
you right, I DRYed up this stuff in 6be8437